src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/SupportedTargetFrameworksEnumProvider.cs

```diff
@@ -52,7 +53,10 @@ public Task<ICollection<IEnumValue>> GetListedValuesAsync()
             {
```

Comment:
Note that the item group containing the target frameworks will have a different name. `SupportedTargetFramework` will still contain the `TargetFrameworkMoniker`s whereas `SupportedTargetFrameworkAlias` is the corresponding item group with the `TargetFramework`s.

Reply:
I think you meant this comment on https://github.com/dotnet/project-system/pull/6607? Yep aware of this of this difference. This PR unblocks being able to pass TargetFramework through.